Brewfile

```diff
@@ -2,12 +2,12 @@ brew "gh"
 brew "ghostscript"
 brew "imagemagick"
 # tap "AdoptOpenJDK/openjdk"
-# cask "adoptopenjdk8"
+# cask "adoptopenjdk"
```

Comment:
do you think we should remove these lines if we are pointing people to use asdf instead?

Reply:
Yea, could do. I left it in case we didn't want to keep it but it's all in version history anyways so might as well. I'll amend and remerge main.